.github/workflows/linux.yml

```diff
@@ -61,6 +61,10 @@ jobs:
         with:
           ref: ${{ env.COMMIT }}
           repository: ${{ env.PR_REPO }}
+      - name: Checkout
+        uses: actions/checkout@v4.1.1
+        with:
+          submodules: true
```

Comment:
Does this workflow checkout Pyccel twice?

Reply:
I think each workflow needs to check out submodules

Reply:
I agree. That is not my question

Reply:
These lines check out more than just submodules